server/sonar-db-dao/src/main/java/org/sonar/core/issue/ShortBranchIssue.java

```diff
@@ -27,20 +28,28 @@
 
 @Immutable
 public class ShortBranchIssue implements Trackable {
+  private final String key;
   private final Integer line;
   private final String message;
   private final String lineHash;
   private final RuleKey ruleKey;
   private final String status;
-  private final String resolution;
+  private final String branchName;
+  private final Date creationDate;
 
-  public ShortBranchIssue(@Nullable Integer line, String message, @Nullable String lineHash, RuleKey ruleKey, String status, @Nullable String resolution) {
+  public ShortBranchIssue(String key, @Nullable Integer line, String message, @Nullable String lineHash, RuleKey ruleKey, String status, String branchName, Date creationDate) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Constructor has 8 parameters, which is greater than 7 authorized. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS00107)